tensorflow/python/eager/context.py

```diff
@@ -1235,6 +1235,31 @@ def invoking_op_callbacks(self):
   def invoking_op_callbacks(self, value):
     self._thread_local_data.invoking_op_callbacks = value
 
+  # Parse PhysicalDevice from the device string.
```

Comment:
nit. `Parse PhysicalDevice from device string` --> `Parse PhysicalDevices from a list of bytes`.

Reply:
done